.github/ISSUE_TEMPLATE/new_resource.yml

```diff
@@ -0,0 +1,73 @@
+name: Request to add a new resource
+description: Use this issue to request to add a new resource to Calitp.org
+title: "New resource request"
+labels: resource
```

Comment:
```suggestion
labels:
  - content
  - resources
```

Updating to match the defined label `resources`, and adding our existing `content` label

![image](https://github.com/cal-itp/calitp.org/assets/1783439/4ae1fd03-2b55-442e-95ee-3323a9d5032a)